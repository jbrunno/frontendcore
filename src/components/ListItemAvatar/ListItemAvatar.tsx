import React from 'react';
import {
  ListItemAvatar as MuiListItemAvatar,
  ListItemAvatarProps as MuiListItemAvatarProps,
} from '@mui/material';

export type ListItemAvatarProps = MuiListItemAvatarProps;

export function ListItemAvatar(props: ListItemAvatarProps) {
  return <MuiListItemAvatar {...props} />;
}
export default ListItemAvatar;
