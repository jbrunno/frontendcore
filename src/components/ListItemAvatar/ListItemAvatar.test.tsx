import React from 'react';
import { render } from '@test-utils/component-testing';
import ListItemAvatar from './ListItemAvatar';

test('renders the ListItemAvatar', () => {
  const { container } = render(<ListItemAvatar />);
  expect(container).toMatchSnapshot();
});
