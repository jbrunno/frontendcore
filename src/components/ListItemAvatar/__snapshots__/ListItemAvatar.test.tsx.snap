// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the ListItemAvatar 1`] = `
.emotion-0 {
  min-width: 56px;
  -webkit-flex-shrink: 0;
  -ms-flex-negative: 0;
  flex-shrink: 0;
}

<div>
  <div
    class="MuiListItemAvatar-root emotion-0"
  />
</div>
`;
